CONTRIBUTING.md

```diff
@@ -37,17 +37,25 @@ Before submitting a PR make sure to run:
 - for formatting
 
   ```shell
-  cargo fmt --all
+  cargo fmt --all -- --check
   ```
 
-- the clippy lints
+- the clippy lints (with warnings treated as errors)
 
   ```shell
-  cargo clippy
+  cargo clippy --all-targets --all -- -D warnings
   ```
 
 - the test suite
 
   ```shell
-  cargo test
+  cargo test --all
   ```
+
+- the lockfile check
+
+  ```shell
+  cargo check --locked --all-targets --all
```

Comment:
Not sure how many times that is actually relevant? Have you run into that failing?

Reply:
hi @sholderbach thanks for the feedback!  I think i was a bit thrown off by the github actions CI calling this command, as parity with CI was the priority for this PR.  It seems like clippy covers this behavior (checking that `Cargo.lock` matches `Cargo.toml`) and so i've edited the instructions with some consolidated commands and clarifying notes.  While I believe this should cover things as far as matching CI, I wonder if some of the steps in the gh workflow can be consolidated?